src/client/pages/ZorgDetail/ZorgDetail.tsx

```diff
@@ -1,22 +1,39 @@
 import { Grid } from '@amsterdam/design-system-react';
-import { InfoDetail } from '../../components';
 import { Datalist } from '../../components/Datalist/Datalist';
 import StatusDetail from '../StatusDetail/StatusDetail';
+import DocumentListV2 from '../../components/DocumentList/DocumentListV2';
+import { capitalizeFirstLetter } from '../../../universal/helpers/text';
 
 export default function ZorgDetail() {
   return (
     <StatusDetail
       stateKey="WMO"
       thema="ZORG"
       pageContent={(isLoading, statusItem) => {
+        let documents = statusItem?.steps?.flatMap((step) => step.documents);
+        const resultaat =
+          statusItem?.resultaat && capitalizeFirstLetter(statusItem?.resultaat);
```

Comment:
Dit type transformaties zijn beter op z'n plek in de service. Je kunt de type WMOVoorzieningFrontend beter uitbreiden met deze gegevens. Je mag de documenten die bij deze service aan de stappen in de statustrein zijn gekoppeld ook weghalen. Let wel op dat je de logica voor welke documenten getoond moeten worden toepast. Zie addDocumentLinksToLineItems. Ik denk dat je deze functie zo moet ombouwen dat je iets kan doen als:

```
...
documents: getDocuments(aanvraag.documents),
...
```